fast64_internal/oot/oot_level.py

```diff
@@ -149,10 +149,17 @@ def onUpdateOOTEmptyType(self, context):
 
 class OOT_ObjectProperties(bpy.types.PropertyGroup):
     version: bpy.props.IntProperty(name="OOT_ObjectProperties Version", default=0)
-    cur_version = 0  # version after property migration
+    cur_version = 1  # version after property migration
 
     scene: bpy.props.PointerProperty(type=OOTSceneProperties)
 
+    @staticmethod
+    def upgrade_changed_props():
+        for obj in bpy.data.objects:
+            if obj.data is None and (obj.ootEmptyType == "Room"):
+                OOTObjectProperty.upgrade_object(obj)
+                obj.fast64.oot.version = obj.fast64.oot.cur_version
```

Comment:
```suggestion
            if obj.data is None and obj.ootEmptyType != "None":
                obj.fast64.oot.version = obj.fast64.oot.cur_version
```
meaning the version of all oot empties is upgraded here, otherwise non-room oot empties would still be "outdated" (their version would be 0 instead of cur_version==1)